server/@types/NewGoalType.ts

```diff
@@ -1,6 +1,9 @@
+import { GoalStatus } from './GoalType'
+
 export type NewGoal = {
-  title: string
-  areaOfNeed: string
+  title?: string
```

Comment:
Doesn't a NewGoal always need a title?

Reply:
the matching API change ensures that there's a submitted title when creating a new one, but basically I wanted to be able to submit a partial NewGoal in order to do updates against a known UUID without having to retrieve and submit all the data again.

Reply:
It might instead be better to use the `Partial<Goal>` type then, or `Partial<NewGoal>`, whichever your fancy

Reply:
done in https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/147/commits/f17ff510a0cf97caeaabf4526cb4a7f6eea6de47